problems/2.functions.js

```diff
@@ -20,17 +20,20 @@ function greeting() {
 * @returns {null}
 * @modifies {global.globalCount}
 */
-function incrementGlobalCount() {
-
+function incrementGlobalCount(val) {
+  let globalCount
+  let global = val
+  return globalCount.push(global)
```

Comment:
We need to reassign the globalCount. We can do this with global.globalCount += 1